### ISSUE TITLE
Failure to start with sync folder on bad disk

### ISSUE THREAD

**Issue by vjoel** [syncthing/syncthing#4381]
A removable disk failed. The disk contained several (but not all) of my sync dirs. This prevents syncthing from starting:

    $ /usr/bin/syncthing -no-browser -no-restart
    [.....] 12:04:24 INFO: syncthing v0.14.38 "Dysprosium Dragonfly" (go1.9 linux-arm64) deb@build.syncthing.net 2017-09-07 09:40:46 UTC [noupgrade]
    [.....] 12:04:24 INFO: My ID: ........
    [.....] 12:04:25 INFO: Single thread SHA256 performance is 20 MB/s using crypto/sha256 (20 MB/s using minio/sha256-simd).
    [.....] 12:04:29 FATAL: failed to upgrade folder marker: remove /media/auto/[...]/.stfolder: no such file or directory

I wonder if it is possible for syncthing to start anyway, marking those folders as inaccessible, but still providing access to the folders that are _not_ on the bad disk?

(A bonus would be flagging the bad disk in syncthing-gtk, but that's probably out of scope.)

**Comment by AudriusButkevicius** [comment]
As the issue template explains, please use the [forum](https://forum.syncthing.net/) for support.
Please use the search on the forum and the closed issue tracker for simillar issues before starting a new thread. 
If this issue is later proven to be a bug, we will reopen this issue or open a new one.

**Comment by AudriusButkevicius** [comment]
You can probably just manually edit the config and remove the folder from there.
We need to do this migration, and if the folder is not available, we can't complete the migration.
